testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -136,6 +136,7 @@
     "Martini_membrane_gro", # for testing the leaflet finder
     "COORDINATES_XTC",
     "COORDINATES_TRR",
+    "COORDINATES_TNG",
```

Comment:
The TNG format is planned to evolve (arbitrary data, new compression algorithms...), a comment saying how the file was produced, and with what version of gromacs would be beneficial I think.